.gitignore

```diff
@@ -31,6 +31,8 @@ node_modules
 
 !typings
 app/**/*.js
```

Comment:
If we're compiling to tmp, then we should change this ignore. We should just ignore everything under `tmp/` and `test/instrumented` that way we don't have to have explicit allows for the js files we're using.